ui/components/app/detected-token/detected-token-selection-popover/detected-token-selection-popover.js

```diff
@@ -47,7 +71,7 @@ const DetectedTokenSelectionPopover = ({
       category: MetaMetricsEventCategory.Wallet,
       properties: {
         source_connection_method: MetaMetricsTokenEventSource.Detected,
-        chain_id: chainId,
+        chain_id: chainIdForMetrics,
```

Comment:
Not sure if this will break metrics if we start sending an array of chain IDs, where it was a string before.  It might be cleaner to add a new parameter for `chain_ids`?

Reply:
That would be something like


```ts
        ...(process.env.PORTFOLIO_VIEW
          ? { chain_ids: chainIds }
          : { chain_id: chainId }),
```